index.md

```diff
@@ -1,26 +1,15 @@
-Anbox Cloud provides a rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud is a cloud platform that containerises mobile workloads using Android as a guest operating system.
```

Comment:
```suggestion
Anbox Cloud makes it possible to run Android apps in the cloud.
```

Reply:
I feel like we miss what `.. for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices` was meant to say which is important to tell.

Reply:
Added back.